pkg/controller.v1beta1/suggestion/suggestion_controller.go

```diff
@@ -47,6 +47,7 @@ import (
 const (
 	ControllerName = "suggestion-controller"
 )
+var ReconcileSuggestionVolumeOverrides = [...] string {"pbt"}
```

Comment:
I think that we can read `ReconcileSuggestionVolumeOverrides` from katib-config so that the users can use this feature for custom suggestion services.
But, in this PR, it would be good to use hardcoded suggestions as the first step for this new feature.

cc: @andreyvelich @johnugeorge @gaocegege

Reply:
Are you thinking we mount the volumes when `persistentVolumeClaimSpec` exists (and is not `{}`) in the katib-config?

Reply:
I think we can add a new field to katib-config, not `persistentVolumeClaimSpec` field.